model/src/main/java/com/example/javaopencommerce/item/Item.java

```diff
@@ -9,11 +9,14 @@
 import com.example.javaopencommerce.image.Image;
 import java.util.List;
 import java.util.stream.Collectors;
+import lombok.AccessLevel;
 import lombok.Builder;
+import lombok.Getter;
 
 @lombok.Value
+@Getter(AccessLevel.NONE)

```

Comment:
Using both Lombok, and having getters in code... This doesn't seem right. It looks like annotation processor will override your getters o fail to do so. Either way it doesn't look good